pxf/HDFSFileDataPXF.html.md.erb

```diff
@@ -2,506 +2,449 @@
+Perform the following steps to create a sample data file, copy the file to HDFS, and use the `HdfsTextSimple` profile to create PXF external tables to query the data:
 
-``` sql
--- writable table with recordkey
-CREATE WRITABLE EXTERNAL TABLE babies_registry (recordkey int, name text, birthday text, weight float)
-  LOCATION ('pxf://namenode_host:51200/babies_1940s'
-            '?ACCESSOR=org.apache.hawq.pxf.plugins.hdfs.SequenceFileAccessor'
-            '&RESOLVER=org.apache.hawq.pxf.plugins.hdfs.WritableResolver'
-            '&DATA-SCHEMA=Babies')
-  FORMAT 'CUSTOM' (formatter='pxfwritable_export');
-INSERT INTO babies_registry VALUES (123456, "James Paul McCartney", "June 18, 1942", 3.800);
-
--- writable table without recordkey
-CREATE WRITABLE EXTERNAL TABLE babies_registry2 (name text, birthday text, weight float)
-  LOCATION ('pxf://namenode_host:51200/babies_1940s'
-            '?ACCESSOR=org.apache.hawq.pxf.plugins.SequenceFileAccessor'
-            '&RESOLVER=org.apache.hawq.pxf.plugins.WritableResolver'
-            '&DATA-SCHEMA=Babies')
-  FORMAT 'CUSTOM' (formatter='pxfwritable_export');
-
--- this record's key will have some default value
-INSERT INTO babies_registry VALUES ("Richard Starkey", "July 7, 1940", 4.0);
-```
+1. Create an HDFS directory for PXF example data files:
 
-The same goes for reading data from an existing file with a key-value format, e.g. a Sequence file.
+    ``` shell
+    $ sudo -u hdfs hdfs dfs -mkdir -p /data/pxf_examples
+    ```
 
-``` sql
--- readable table with recordkey
-CREATE EXTERNAL TABLE babies_1940 (recordkey int, name text, birthday text, weight float)
-  LOCATION ('pxf://namenode_host:51200/babies_1940s'
-            '?FRAGMENTER=org.apache.hawq.pxf.plugins.hdfs.HdfsDataFragmenter'
-            '&ACCESSOR=org.apache.hawq.pxf.plugins.hdfs.SequenceFileAccessor'
-            '&RESOLVER=org.apache.hawq.pxf.plugins.hdfs.WritableResolver'
-            '&DATA-SCHEMA=Babies')
-  FORMAT 'CUSTOM' (formatter='pxfwritable_import');
--- retrieve each record's key
-SELECT * FROM babies_1940;
-
--- readable table without recordkey
-CREATE EXTERNAL TABLE babies_1940_2 (name text, birthday text, weight float)
-  LOCATION ('pxf://namenode_host:51200/babies_1940s'
-            '?FRAGMENTER=org.apache.hawq.pxf.plugins.hdfs.HdfsDataFragmenter'
-            '&ACCESSOR=org.apache.hawq.pxf.plugins.hdfs.SequenceFileAccessor'
-            '&RESOLVER=org.apache.hawq.pxf.plugins.hdfs.WritableResolver'
-            '&DATA-SCHEMA=Babies') 
-  FORMAT 'CUSTOM' (formatter='pxfwritable_import');
--- ignores the records' key
-SELECT * FROM babies_1940_2; 
-```
+2. Create a delimited plain text data file named `pxf_hdfs_simple.txt`:
+
+    ``` shell
+    $ echo 'Prague,Jan,101,4875.33
+Rome,Mar,87,1557.39
+Bangalore,May,317,8936.99
+Beijing,Jul,411,11600.67' >> pxf_hdfs_simple.txt
```

Comment:
Maybe change the redirect operator to `>` otherwise if a user already has a `pxf_hdfs_simple.txt` file for whatever reason the above will append to it